smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/PersistedQueryIT.java

```diff
@@ -20,23 +20,30 @@
 import com.adobe.cq.testing.junit.rules.CQPublishClassRule;
 import com.adobe.cq.testing.junit.rules.CQRule;
 import org.apache.sling.testing.clients.ClientException;
+import org.apache.sling.testing.clients.util.poller.Polling;
 import org.junit.BeforeClass;
 import org.junit.ClassRule;
 import org.junit.Rule;
 import org.junit.Test;
 
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
 public class PersistedQueryIT {
 
+    private static final long TIMEOUT = TimeUnit.SECONDS.toMillis(30);
+    private static final long DELAY = TimeUnit.SECONDS.toMillis(1);
+
     @ClassRule
     public static final CQPublishClassRule cqBaseClassRule = new CQPublishClassRule();
-    static CQClient adminPublish;
+    static CQClient anonymous;
 
     @Rule
     public CQRule cqBaseRule = new CQRule(cqBaseClassRule.publishRule);
 
     @BeforeClass
     public static void beforeClass() {
-        adminPublish = cqBaseClassRule.publishRule.getAdminClient(CQClient.class);
+        anonymous = cqBaseClassRule.publishRule.getClient(CQClient.class, null, null);
```

Comment:
@andresbott pointed out that anonymous client is most likely is causing a NPE and asked to double check the problem. I performed the checking and can confirm the issue:

Use of anomyous client `anonymous = cqBaseClassRule.publishRule.getClient(CQClient.class, null, null)` is causing the test to fail:
```
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 10.721 s <<< FAILURE! - in com.adobe.cq.cloud.testing.it.smoke.PersistedQueryIT
[ERROR] testPersistedQueryEndpointAccessible(com.adobe.cq.cloud.testing.it.smoke.PersistedQueryIT)  Time elapsed: 10.721 s  <<< ERROR!
java.util.concurrent.TimeoutException: Call failed to return true in 10000 ms. Last exception was: java.lang.NullPointerException at com.adobe.cq.cloud.testing.it.smoke.PersistedQueryIT.testPersistedQueryEndpointAccessible(PersistedQueryIT.java:63)
```
when authenticated client is used `adminPublish = cqBaseClassRule.publishRule.getAdminClient(CQClient.class)` same test passes.

Tested with:
```
mvn -Peaas-local clean verify \
-Dcloud.author.url=https://author-p87056-e199751-cmstg.adobeaemcloud.com/ \
-Dcloud.author.user=admin \
-Dcloud.author.password='~r<x]N40M}tSy@Qr?2@+l-C@' \
-Dcloud.publish.url=https://publish-p87056-e199751-cmstg.adobeaemcloud.com/ \
-Dcloud.publish.user=admin \
-Dcloud.publish.password='~r<x]N40M}tSy@Qr?2@+l-C@' \
```

Reply:
done